fastir/common/file_info.py

```diff
@@ -0,0 +1,100 @@
+import hashlib
+import filetype
+from datetime import datetime
+
+from pefile import PE
+
+from .logging import logger
+
+
+MAX_PE_SIZE = 50 * 1024 * 1024
+
+
+class FileInfo:
+    def __init__(self, path_object):
+        self._path_object = path_object
+        self.size = path_object.get_size()
+
+        self._info = {}
+        self._content = b""
+
+    def compute(self):
+        self.md5 = hashlib.md5()
+        self.sha1 = hashlib.sha1()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of insecure MD2, MD4, MD5, or SHA1 hash function.](https://app.codacy.com/manual/muller.gael/fastir_artifacts/pullRequest?prid=5498175)